lib/theme-options.php

```diff
@@ -104,6 +104,21 @@ function add_options_page_metabox() {
 		// Set our CMB2 fields
 
 		$cmb->add_field( array(
+			'name' => __( 'Max age of popular posts', 'IGV' ),
```

Comment:
we coooould allow them to put in the string themselves. Its text to string to `1 week`, `3 months`, `5 years` would all be fine. But yeah could change to weeks so they can do like 6 or something